data/docs/gcp-monitoring/gke-tracing.mdx

```diff
@@ -0,0 +1,1190 @@
+<i>
+Application-level traces in SigNoz
+</i>
+</figcaption>
+</figure>
+
+
+## Troubleshooting
+
+If you encounter any issues while setting up logging and metrics for your GKE cluster, follow these troubleshooting steps:
+
+1. **Check the logs of the OpenTelemetry Collector:**
+
+    ```sh
+    kubectl logs -f -n signoz -l app.kubernetes.io/component=otel-agent
+    ```
+
+    Review the logs for any error messages or indications of misconfiguration.
+
+2.  **Verify the rendered configuration:**
+    ```sh
+    kubectl get configmap/kubelet-otel-k8s-infra-otel-agent -n signoz -o yaml
+    ```
+    Ensure the configuration matches your expected settings, including the SigNoz API key and the OpenTelemetry Collector endpoint.
+
+3. **Confirm that the necessary Kubernetes resources are created:**
+    ```sh
+    kubectl get pods,services,configmaps -n signoz
+    ```
+    _Check if the required pods, services, and config maps are running and in a healthy state._
+
+4. **Verify network connectivity:**
+
+    * Ensure that the GKE cluster has network access to the SigNoz ingestion endpoint (`ingest.<region>.signoz.cloud:443`).
+    * Check if there are any network security groups or firewalls blocking the required ports.
+
+5. **Double-check the SigNoz API key:**
+
+    * Confirm that the provided signozApiKey is correct and has the necessary permissions to ingest data.
+
+6. **Check if all the Pixie deployments are in the correct order:**
+    * Check all the configurations and deployments are in the correct order so that Pixie can forward all the scraped traces to the respective endpoint.
+
+</TabItem>
+
+<TabItem value="self-host" label="Self-Host">
+
+## Prerequisites
+
+* A [GKE cluster](https://cloud.google.com/kubernetes-engine) 
+* Install [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) to access the GKE cluster.
+* [Install Helm](https://helm.sh/docs/intro/install/)
+* [Self-Hosted SigNoz in GKE Cluster](https://signoz.io/docs/install/kubernetes/gcp/) ( If you are hosting SigNoz in Google Kubernetes Engine (GKE), you should refer to the specific guide that outlines the minimum requirements and provides details for the override-values.yaml ) OR if you’re hosting SigNoz on VM, follow [Self-Hosted SigNoz on VM ](https://signoz.io/docs/install/docker/).
+
+## Quick Start
+
+Check that the nodes in your GKE clusters are in `Ready` state:
+
+<figure data-zoomable align="left">
+    <img
+        src="/img/docs/gcp-monitoring/gke-tracing/cluster-status.webp"
+        alt="GKE Cluster Status"
+    />
+<figcaption>
+<i>
+GKE Cluster Status
+</i>
+</figcaption>
+</figure>
+
+**Step 1:** Add the SigNoz helm repo using this command
+
+```sh
+helm repo add SigNoz [https://charts.signoz.io](https://charts.signoz.io)
+```
+
+<Tabs>
+<TabItem value="self-hosted-on-gke-cluster" label="For Self-hosted SigNoz in GKE Cluster" default>
+
+**Step 2:** Install the OTel Collector agent in GKE where SigNoz is running in a Self-hosted VM.
```

Comment:
Should be : Install the OTel Collector agent in GKE where SigNoz is running in a GKE cluster

Reply:
will fix it